homework/Data warehousing homework.py

```diff
@@ -0,0 +1,156 @@
+os.listdir("/Users/chloemaine/Documents/Chloe/BGSE/Computing/warehousing-operational-databases/homework/extracts")
+
+#Open CSVs
+products = pd.read_csv("products.csv")
+orders = pd.read_csv("orders.csv")
+employees = pd.read_csv("employees.csv")
+
+#Create sub-tables
+customers = orders[['customer_number', 'customer_name', 'contact_last_name', 'contact_first_name', 'city', 'state', 'country', 'sales_rep_employee_number', 'credit_limit', 'customer_location']].copy()
+customers_clean = customers.drop_duplicates()
+orders_clean = orders[['order_number', 'customer_number', 'product_code', 'quantity_ordered', 'price_each', 'order_line_number', 'order_date', 'required_date', 'shipped_date', 'status', 'comments']]
+orders_composition_clean = orders_clean[['order_number', 'customer_number', 'product_code', 'quantity_ordered', 'price_each', 'order_line_number']]
+orders_composition_clean_denulled=orders_composition_clean.astype(object).where(pd.notnull(orders_composition_clean), None)
+offices_clean = employees[['office_code', 'city', 'state', 'office_location', 'country']].drop_duplicates()
+employees_clean = employees [['employee_number','office_code', 'employee_number', 'last_name', 'first_name', 'reports_to', 'job_title']]
+product_stock_details_clean =products[['product_code', 'product_name', 'quantity_in_stock', 'product_description', 'buy_price', '_m_s_r_p']].drop_duplicates()
+product_stock_details_clean_denulled=product_stock_details_clean.astype(object).where(pd.notnull(product_stock_details_clean), None)
+products_clean = products[['product_code', 'product_line', 'product_scale', 'product_vendor', 'html_description']].drop_duplicates()
+products_clean_denulled=products_clean.astype(object).where(pd.notnull(product_stock_details_clean), None)
+products_stock_details_clean_copy= product_stock_details_clean.copy()
+orders_delivery_clean = orders_clean[['order_number','order_date', 'required_date', 'shipped_date','price_each', 'status', 'comments']].drop_duplicates()
+orders_delivery_clean_denulled= orders_delivery_clean.astype(object).where(pd.notnull(orders_delivery_clean), None)
+orders_delivery_clean_denulled['order_delivery_id']= np.arange(len(orders_delivery_clean_denulled))
+
+#Install psycopg2 package
+import dataset
+
+ds = dataset.connect("postgresql://postgres@localhost/my_first_database")
+
+cs = ds['customers_1']
+cs.insert_many(customers_clean.to_dict('records'))
+
+e1 = ds['employees_1']
+e1.insert_many(employees_clean.to_dict('records'))
+
+of1 = ds['offices_1']
+of1.insert_many(offices_clean.to_dict('records'))
+
+oc1 = ds['orders_composition_1']
+oc1.insert_many(orders_composition_clean_denulled.to_dict('records'))
+
+or1 = ds['orders_delivery_1']
+or1.insert_many(orders_delivery_clean_denulled.to_dict('records'))
+
+psd1 = ds['product_stock_details_1']
+psd1.insert_many(product_stock_details_clean_denulled.to_dict('records'))
+
+pr1 = ds['products_1']
+pr1.insert_many(products_clean.to_dict('records'))
+
+
+
+
+#WIP code
```

Comment:
Clean this up! This is what git is for. You can leave the "old versions" as previous commits, so you can always go back to them, but don't leave them hanging around in comments. Keep things nice and tidy!